frontend/src/tests/mocks/dataMocks.js

```diff
@@ -277,6 +291,29 @@ export const STATUS_OPTIONS = {
   ]
 };
 
+export const REGION_OPTIONS = { 
+  'options': [{
```

Comment:
Any reason for using single quotes on these keys? They shouldn't be needed, and I don't think we're generally adding them in other files

Reply:
single quotes are removed, these are my mock objects, Usually copy and pasted from somewhere, will pay more attention to consistency!!

Reply:
also converted all the values to double quotes